feat(python): Improve integrations page (#10575)
getsentry/sentry-docs: docs/platforms/python/configuration/options.mdx

Environment variable: `SENTRY_DEBUG`

> Turns debug mode on or off. If `debug` is enabled, the SDK will attempt to print out debugging information. This can be useful for example if something goes wrong with event sending. The default is always `False`. It's generally not recommended to turn it on in production because of the increase in log output.

Reviewer: ```suggestion
Turns debug mode on or off. When `True`, the SDK will attempt to print out debugging information. This can be useful if something goes wrong with event sending. The default is always `False`. It's generally not recommended to turn it on in production because of the increase in log output.
```

> Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is always `False`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any safety concerns.

Reviewer 2: > will not cause any safety concerns

I realize your change is not really affecting this part of the line, but what are we even referring to here? I also wonder whether it is wise for our docs page to make such a blanket claim about `debug=True` being completely safe – like I feel this statement could open Sentry to legal liability if something did go wrong when someone was using `debug=True`.

I would propose we remove this statement, if we want to keep it we should definitely specify what exactly we mean and run the wording by legal to make sure this does not increase liability risk

Reviewer 2: Reword looks good to me!

<ConfigKey name="integrations" />

> Fine-tune the integrations to enable. See [the integrations page](/platforms/python/integrations) for examples.

Author: `List of integrations to enable` is a bit misleading though. It could be understood to imply that this overrides the logic that auto-enables integrations. Maybe:

> List of integrations to enable in addition to auto-enabling integrations (link to integrations page). This setting can be used to override the default config options for a specific auto-enabling integration or to add an integration that is not auto-enabled.

> Fine-tune the integrations to enable. See [the integrations page](/platforms/python/integrations) for examples.

Reviewer 2: Yeah, I like your suggestion!
